docs/src/main/docs/health/01_health.adoc

```diff
@@ -241,3 +243,12 @@ Routing.builder()
     ]
 }
 ----
+
+=== Strict JSON Output
+
+The JSON responses shown above contain properties `"status"` and `"outcome"` with the same
+values. Helidon reports both of these to maintain backward compatibility with older
+versions of the MicroProfile Metrics library. This behavior can be disabled by setting
+the property `health.backward-compatible` to `false`, in which case only `"status"`
+is reported. Future versions of Helidon will drop support for older versions of Metrics,
```

Comment:
Same: health, not metrics?

Reply:
Yes, I don't know why I get the two mixed up all the time LOL